pulp_container/app/migrations/0038_add_manifest_metadata_fields.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.2.10 on 2024-02-29 16:04
+import warnings
+
+from django.db import migrations, models
+
+
+def print_warning_for_initializing_image_nature(apps, schema_editor):
+    warnings.warn(
+        "Run 'pulpcore-manager init-image-nature' to initialize metadata (i.e., annotations "
+        "and labels) for all manifests. The metadata are stored in associated artifacts and "
```

Comment:
metadata is singular, hence is stored

Reply:
I would remove last sentence, what kind of important info it brings? "their contents should be accessible through the Manifest model now" what contents? all contents? or just selected ones.
Users are not interested in tech.details of how a manifests is composed, imo

Reply:
a shorter concise version: "Run 'pulpcore-manager init-image-nature' to initialize and expose metadata (i.e., annotations and labels) for the manifests"

Reply:
I wanted to be explicit when it comes to reading data from the storage back-end. If administrators run this migration, they might be a bit surprised by the the rise in expenses.

Reply:
the migration itself does not read from disk anything, it is the django management command that it does

Reply:
I meant the django-management command migration. We are getting lost in this terminology because we always talk about migrating data, xixi.